src/user-icon/user-icon.scss

```diff
@@ -52,6 +52,7 @@
   }
 
   > .iui-icon {
```

Comment:
We do not have this case in react, do we...

Reply:
Correct